src/model/Ressource.ts

```diff
@@ -73,14 +73,15 @@ export default abstract class Ressource {
   protected _queryUrl?: string;
 
   private data?: APIObject;
+  key_id?: string
```

Comment:
Not sure we need to do that, because the `key_id` is a property specific to `SshKey` and we have already this property on the [model](https://github.com/platformsh/platformsh-client-js/blob/master/src/model/SshKey.ts#L19)